contracts/plugins/lido/LidoStakedEthCollateral.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "contracts/plugins/assets/AppreciatingFiatCollateral.sol";
+import "contracts/plugins/lido/IWSTETH.sol";
+import "contracts/plugins/assets/OracleLib.sol";
+import "contracts/libraries/Fixed.sol";
+
+/**
+ * @title Lido Staked Eth Collateral
+ * @notice Collateral plugin for Lido stETH,
+ * tok = wstETH  (wrapped stETH)
+ * ref = stETH (pegged to ETH 1:1)
+ * tar = ETH
+ * UoA = USD
+ */
+contract LidoStakedEthCollateral is AppreciatingFiatCollateral {
+    using OracleLib for AggregatorV3Interface;
+    using FixLib for uint192;
+
+    AggregatorV3Interface public immutable targetUnitChainlinkFeed; // {UoA/target}
+    uint48 public immutable targetUnitOracleTimeout; // {s}
+
+    constructor(
+        CollateralConfig memory config,
+        uint192 revenueHiding,
+        AggregatorV3Interface targetUnitChainlinkFeed_,
+        uint48 targetUnitOracleTimeout_
+    ) AppreciatingFiatCollateral(config, revenueHiding) {
+        require(address(targetUnitChainlinkFeed_) != address(0), "missing targetUnit feed");
+        require(targetUnitOracleTimeout_ > 0, "targetUnitOracleTimeout zero");
+        targetUnitChainlinkFeed = targetUnitChainlinkFeed_;
+        targetUnitOracleTimeout = targetUnitOracleTimeout_;
+        exposedReferencePrice = _underlyingRefPerTok().mul(revenueShowing);
+    }
+
+    /// Can revert, used by other contract functions in order to catch errors
+    /// @param low {UoA/tok} The low price estimate
+    /// @param high {UoA/tok} The high price estimate
+    /// @param pegPrice {target/ref}
+    function tryPrice()
+        external
+        view
+        override
+        returns (
+            uint192 low,
+            uint192 high,
+            uint192 pegPrice
+        )
+    {
+        // Get current market peg {eth/steth}, but the intended {target/ref} will be returned
+        pegPrice = chainlinkFeed.price(oracleTimeout); // {target/ref}
```

Comment:
not something not necessarily change in this pr, just dropping a thought....

i don't think we've standardized what `chainlinkFeed` means across collateral types...sometimes it is used as UoA/target, sometimes it is used as target/ref.  I think it might be worth standardizing and renaming so that we can be clear about what it represents, as it could lead to confusion down the road otherwise.

Reply:
Ive looked at the ones we have that use additional feeds for example the NonFiatcollateral (wBTC) and in that case the default one is the peg, and the additional is the Target feed. So I did it in that same way. Buy I can revert them if preferred. (I think we should revert them in wBTC as well if we do that probably).

Reply:
We should definitely sync them all and be consistent. Lets have this discussion before getting any of these to master next week.